anypoint-signin-aware.js

```diff
@@ -450,42 +399,58 @@ export const AnypointAuth = {
 };
 
 /**
- * `<anypoint-signin-aware>` is used to authenticate the user in Anypoint
- * core services API.
+ * `<anypoint-signin-aware>` is used to authenticate the user with the Anypoint Platform.
  *
  * The `anypoint-signin-aware-success` event is triggered when a user
  * successfully authenticates. It also sets `accessToken` property that can be
- * used to interact with Anypoint APIs.
+ * used to interact with Anypoint Platform APIs.
+ *
  * The `anypoint-signin-aware-signed-out` event is triggered when a user
  * signs out via calling `signOut()` function.
  *
+ * The `oauth2-token-response` event is triggered when a user
+ * successfully authenticates for the authorization_code flow via the /authorize endpoint. An authorization code
+ * is returned in the event and should be used to exchange for an access token via the /token endpoint from
+ * your backend server.
+ *
  * You can bind to `signedIn` property to monitor authorization state.
  * ##### Example
  *
  *     <anypoint-signin-aware signed-in="{{isSigned}}"></anypoint-signin-aware>
  *
  * The `clientId` and `redirectUri` properties has to be set before using the
  * component. `clientId` and associated with it `redirectUri` has to be set up
- * with Anypoint authorization server. Contact Anypoint Core services for
- * more information.
+ * with Anypoint authorization server.
+ * Contact Anypoint Access Management for more information.
  *
  * ##### Example
  *
- *     <anypoint-signin-aware
- *      client-id="abc123"
- *      redirect-uri="https://auth.domain.com/oauth2/redirect"></anypoint-signin-aware>
+ *      <anypoint-signin-aware
```

Comment:
This feels like a bad name. Signin-aware is an adjective. It feels like we need a noun here as a name.